great_expectations_cloud/agent/agent.py

```diff
@@ -444,20 +448,24 @@ def _create_scheduled_job_and_set_started(self, event_context: EventContext) ->
         Args:
             event_context: event with related properties and actions.
         """
+        org_id = self.get_organization_id()
         data = {
             "correlation_id": event_context.correlation_id,
             "event": event_context.event.dict(),
         }
-        LOGGER.info("Creating scheduled job and setting started", extra=data)
+        LOGGER.info(
+            "Creating scheduled job and setting started", extra={**data, "organization_id": org_id}
+        )
 
         agent_sessions_url = (
-            f"{self._config.gx_cloud_base_url}/organizations/{self.get_organization_id(event_context)}"
-            + "/agent-jobs"
+            f"{self._config.gx_cloud_base_url}/organizations/{org_id}" + "/agent-jobs"
         )
         with create_session(access_token=self.get_auth_key()) as session:
             payload = Payload(data=data)
             session.post(agent_sessions_url, data=payload.json())
-            LOGGER.info("Created scheduled job and set started", extra=data)
+            LOGGER.info(
+                "Created scheduled job and set started", extra={**data, "organization_id": org_id}
```

Comment:
```suggestion
                "Created scheduled job and set started", extra={**data, "organization_id": str(org_id)}
```

Reply:
This `**data` might actually barf in `logging` if values arent serializable

Reply:
Also a bit risky to log without knowing sensitivity of data

Reply:
Thanks!